reference/tools/data-migration/manage-tasks.md

```diff
@@ -8,9 +8,11 @@ category: reference
 
 This document describes how to manage and maintain the data replication task using the [dmctl](/reference/tools/data-migration/overview.md#dmctl) component. For the Data Migration cluster deployed using DM-Ansible, the dmctl binary file is in `dm-ansible/dmctl`.
 
-## dmctl basic usage
+The dmctl component supports the interactive mode for manual operations, and also supports the command mode for the script.
 
-This section shows the basic usage of dmctl commands.
+## dmctl interactive mode
+
+This section described the basic use of dmctl commands in the interactive mode.
```

Comment:
@toutdesuite Why use the past tense?

Reply:
@lilin90 The mistake is resolved in https://github.com/pingcap/docs/pull/2118. Thanks for pointing it out.